hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/MergeIntoHoodieTableCommand.scala

```diff
@@ -742,7 +744,10 @@ case class MergeIntoHoodieTableCommand(mergeInto: MergeIntoTable) extends Hoodie
       HoodieSyncConfig.META_SYNC_PARTITION_EXTRACTOR_CLASS.key -> hiveSyncConfig.getString(HoodieSyncConfig.META_SYNC_PARTITION_EXTRACTOR_CLASS),
       SqlKeyGenerator.PARTITION_SCHEMA -> partitionSchema.toDDL,
       PAYLOAD_CLASS_NAME.key -> classOf[ExpressionPayload].getCanonicalName,
+      HoodieTableConfig.PAYLOAD_CLASS_NAME.key() -> classOf[ExpressionPayload].getCanonicalName,
```

Comment:
Should this be avoided?

Reply:
You are asking if I should use the write config instead of table config? 
I just got rid of it and the MIT tests seem like they are running fine so I pushed the change

Reply:
Yeah, just the write config to avoid the risk of overwriting the table config.